src/test/ui/traits/cycle-cache-err-60010.stderr

```diff
@@ -6,22 +6,6 @@ LL |     _parse: <ParseQuery as Query<RootDatabase>>::Data,
    |
    = note: required because of the requirements on the impl of `Query<RootDatabase>` for `ParseQuery`
 
-error[E0275]: overflow evaluating the requirement `Runtime<RootDatabase>: std::panic::RefUnwindSafe`
-  --> $DIR/cycle-cache-err-60010.rs:31:5
-   |
-LL |     type Storage;
-   |          ------- associated type defined here
-...
-LL | impl Database for RootDatabase {
-   | ------------------------------ in this `impl` item
-LL |     type Storage = SalsaStorage;
-   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-   |
-   = note: required because it appears within the type `RootDatabase`
-   = note: required because of the requirements on the impl of `SourceDatabase` for `RootDatabase`
-   = note: required because of the requirements on the impl of `Query<RootDatabase>` for `ParseQuery`
-   = note: required because it appears within the type `SalsaStorage`
-
-error: aborting due to 2 previous errors
```

Comment:
What happened here? How did your change affect this‽

Reply:
It might be a parallel-compiler thing... Let me try disabling it and see if it makes a difference.

Reply:
Yeah it was; all fixed now.